tier1/checklist.md

```diff
@@ -0,0 +1,368 @@
+  - Can it be removed? Is it absolutely needed to function? Can it be shipped with synthetic data instead?
+ - Does the code interface with any CMS' internal-only systems(e.g. mainframes,JIRA instances, databases, etc..)?
+ - Does the repository contain any keys or credentials to access or authenticate with CMS' systems?
+   - Can it be removed or is it needed?
+
+   If you answered "yes" to any of the above  questions, your project may be 'sensitive' in nature, and require a more thorough review before sharing publicly. Please reach out to opensource@cms.hhs.gov for guidance. If you answer yes to any of these questions above, it is best to seek quidance **before** releasing open source.
+
+   **Results** 
+
+   *Insert Review Here*
+
+
+
+
+   ---
+
+
+
+
+   ### Code Review
+
+   The existing codebase should be given a one time, top-to-bottom code quality and security vulnerability review by two (or more) engineers who have written production code within the path two years, in the languages used in the project.Engineers should review credential management practices with the development team to ensure that any keys, passwords, or other sensitive configurations are not checked into source code or in the git history.
+
+   The engineers can be federal government employees or trusted partners from outside the agency from other contracts, or form independent testing contracts.Their names, organizations, comments and approval/disapproval on the overall codebase should be tracked in the documents.
+
+   To provide independent review, ideally the engineers should not have been involved in the development of the software product.This includes engineers who wrote part of the software or who direclty provided technical direction and oversight in the creation of the software.
+
+   As part of the code review, engineers should reference modern listings of the most significant software security vulnerabilities. For instance, an acceptable description would be that the engineers showed how they used automated tools and manual review to check each item in __OWASP's current 10 Most Critical Web Application Security Risks.__
+
+
+
+
+   **Results**
+
+   *Insert Review Here*
+
+
+   ---
+
+
+   ### Code Analysis 
+
+   A best practice and a requirement for releasing a repository open source at many parts of CMS is to run at least one automated tool for code analysis (such as static code analysis,repolinters, secret scanners) on the codebase to detect for security vulnerabilities or sensitive information, and results have been appropriately addressed. Even if all findings are eventually fixed, if the initial scans revealed significant, severe vulnerabilities(such as SQL injection vulnerabilities),this may indicate that the software development team was not adhering to the best practices required for open source public release, and should be given additional review.
```

Comment:
add space between "code analysis," and "repolinters", space between "ties)," and "this",